pkg/minikube/driver/driver.go

```diff
@@ -143,7 +143,7 @@ func BareMetal(name string) bool {
 
 // NeedsRoot returns true if driver needs to run with root privileges
 func NeedsRoot(name string) bool {
```

Comment:
I think we can get rid of this function and all calls to it entirely -- pretty sure only the none driver needs roots so this isn't useful anymore with this PR :)

Reply:
I removed that function with a new commit.
Once everything is fine I will squash everything before final merge.